app/javascript/dashboard/mixins/time.js

```diff
@@ -12,5 +12,29 @@ export default {
       const unixTime = fromUnixTime(time);
       return formatDistanceToNow(unixTime, { addSuffix: true });
     },
+    shortTimestamp(time) {
```

Comment:
Can we write tests for this? Correct me if I am wrong, I am assuming the input to this function comes from date-fns distanceInWords method, so if we change the underlying library this could break